test/inductor/test_auto_functionalize.py

```diff
@@ -181,11 +181,9 @@ def f(x, y, z, n):
                 self.assertExpectedInline(
                     post_grad_graphs,
                     """\
-def forward(self, arg0_1: "f32[3][1]cpu", arg1_1: "f32[3][1]cpu", arg2_1: "f32[3][1]cpu", arg3_1: \
-"f32[3][1]cpu", arg4_1: "f32[3][1]cpu"):
+def forward(self, arg0_1: "f32[3][1]cpu", arg1_1: "f32[3][1]cpu", arg2_1: "f32[3][1]cpu", arg3_1: "f32[3][1]cpu", arg4_1: "f32[3][1]cpu"):
         # No stacktrace found for following nodes
-        foo_default = torch.ops.mylib.foo.default(arg4_1, [arg2_1, arg3_1], arg1_1, 2, arg0_1);  arg4_1 = arg2_1 = \
-arg3_1 = arg1_1 = arg0_1 = foo_default = None
+        foo_default = torch.ops.mylib.foo.default(arg0_1, [arg3_1, arg4_1], arg1_1, 2, arg2_1);  arg0_1 = arg3_1 = arg4_1 = arg1_1 = arg2_1 = foo_default = None
```

Comment:
Why are these changing?

Reply:
I figured this out - basically since I added a check that a func is in get_overridable_functions first, we now don't always check the tensor args for subclasses. As a result, the lazy variables are no longer evaluated when they were before, hence a different order.

Reply:
I'm confused because torch.ops.higher_order.auto_functionalized *is* torch_function overrideable in eager-mode, so we should be testing the tensor args for subclasses, so there should be no divergence from the previous graph

Reply:
Hmm I put a breakpoint, it looks like torch.ops.mylib.foo is not in the set, is that expected?

Reply:
Looks like we found one of the divergences. Everything under torch.ops.* is torch_function overrideable, but does not appear in the get_overrides list.

Reply:
For this one we can probably just hardcode torch.ops.blah since adding those to the list seems tricky (torch.ops.blah is dynamically populated)

Reply:
Yeah I think I'll hardcode this, thanks for the suggestion.

Reply:
Updated